tsdb/chunks/chunks.go

```diff
@@ -67,6 +67,20 @@ type Meta struct {
 	MinTime, MaxTime int64
 }
 
+// Iterator iterates over the chunk of a time series.
+type Iterator interface {
+	//// Seek advances the iterator forward to the given timestamp.
+	//// It advances to the chunk with min time at t or first chunk with min time after t.
+	//Seek(t int64) bool
+	// At returns the current meta.
+	// It depends on implementation if the chunk is populated or not.
```

Comment:
why is that? Why not just return non empty chunks?

Reply:
I don't think you know what I mean here. Populated is if you have only `Ref` or if you have actually `Bytes` of chunk there (: